src/frontend/classic/components/MediaSelect.svelte

```diff
@@ -49,13 +50,16 @@
 
     let mediaNameFilter = "";
     $: {
-        filteredmedias = medias.filter((item) => {
-            return (
-                item?.Parent?.Title?.toLowerCase().indexOf(
-                    mediaNameFilter.toLowerCase()
-                ) !== -1
-            );
-        });
+        if (mediaNameFilter === "") filteredmedias = medias;
+        else {
+          const fuse = new Fuse(medias, {
+            keys: ["Title"],
+            findAllMatches: true,
+            ignoreLocation: true,
+            minMatchCharLength: 1,
+          });
+          filteredmedias = fuse.search(mediaNameFilter).map((item) => item.item);
```

Comment:
Any assumptions about performance when searching within 750~1000 media items while typing in the search box?

Reply:
in worst case .... DEBOUCE !
https://svelte.dev/repl/f55e23d0bf4b43b1a221cf8b88ef9904?version=3.12.1
or with lodash: https://svelte.dev/repl/52e9b3d7a5654021b58597fe807d53d8?version=3.20.1

Reply:
Performance has been measured by using the following code section:

```typescript
    $: {
      console.log(window.performance.now());
      filteredmedias = (mediaNameFilter === "") ? medias : fuse.search(mediaNameFilter).map((item) => item.item);
      console.log(window.performance.now());
    }
```

Processing timings observed for my current setup are around 4.2 milliseconds per keystroke for 749 media items, with no difference for longer search terms. This would ensure the filter reacts on time for ~238 keystrokes per second. If we suppose linearity on the case (**which might not be real after all**), a 50,000 media items query would suppose a 66.75x slow down for the processing, showing response at 280.35 milliseconds per keystroke.
I would suggest that we set a testbench for this and use a command queue somehow. The debounce timer-based solution proposed by @Sheepux seems feasible as well, but this feature should be activated only when performance issues be detected.

<details><summary>See CPU hardware specs for the test</summary>

```text
Architecture:                    x86_64
CPU op-mode(s):                  32-bit, 64-bit
Byte Order:                      Little Endian
Address sizes:                   43 bits physical, 48 bits virtual
CPU(s):                          16
On-line CPU(s) list:             0-15
Thread(s) per core:              2
Core(s) per socket:              8
Socket(s):                       1
NUMA node(s):                    1
Vendor ID:                       AuthenticAMD
CPU family:                      23
Model:                           113
Model name:                      AMD Ryzen 7 3800XT 8-Core Processor
Stepping:                        0
Frequency boost:                 enabled
CPU MHz:                         2195.377
CPU max MHz:                     5575.7808
CPU min MHz:                     2200.0000
BogoMIPS:                        7785.90
Virtualization:                  AMD-V
L1d cache:                       256 KiB
L1i cache:                       256 KiB
L2 cache:                        4 MiB
L3 cache:                        32 MiB
NUMA node0 CPU(s):               0-15
Vulnerability Itlb multihit:     Not affected
Vulnerability L1tf:              Not affected
Vulnerability Mds:               Not affected
Vulnerability Meltdown:          Not affected
Vulnerability Spec store bypass: Mitigation; Speculative Store Bypass disabled via prctl and seccomp
Vulnerability Spectre v1:        Mitigation; usercopy/swapgs barriers and __user pointer sanitization
Vulnerability Spectre v2:        Mitigation; Full AMD retpoline, IBPB conditional, STIBP conditional, RSB filling
Vulnerability Srbds:             Not affected
Vulnerability Tsx async abort:   Not affected
Flags:                           fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx mmxext fxsr_opt pdpe1gb rdtscp lm constant_tsc
                                  rep_good nopl nonstop_tsc cpuid extd_apicid aperfmperf pni pclmulqdq monitor ssse3 fma cx16 sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand lahf_lm cmp_
                                 legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs skinit wdt tce topoext perfctr_core perfctr_nb bpext perfctr_llc mwaitx cpb cat_l3 
                                 cdp_l3 hw_pstate sme ssbd mba sev ibpb stibp vmmcall fsgsbase bmi1 avx2 smep bmi2 cqm rdt_a rdseed adx smap clflushopt clwb sha_ni xsaveopt xsavec xgetbv1 xsa
                                 ves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local clzero irperf xsaveerptr rdpru wbnoinvd arat npt lbrv svm_lock nrip_save tsc_scale vmcb_clean flushbyasi
                                 d decodeassists pausefilter pfthreshold avic v_vmsave_vmload vgif umip rdpid overflow_recov succor smca
```
</details>

Reply:
I'll create Sheep's Fat Mangas so that we can test it. I have mangahere list on local cache and will upload it.
However i think we can just improve it later and merge this PR

Reply:
new connector available :)